index.js

```diff
@@ -1,88 +1,123 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+counter = 1
```

Comment:
I would get into the habit of using `let` or `const` whenever you declare a variable!
If you don't declare a variable using a keyword, it becomes a global variable in JS, which can potentially lead to bugs down the road!